app/Nova/Metrics/ActiveStudentsInMajor.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Metrics;
+
+use App\Major;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Laravel\Nova\Metrics\Value;
+use Laravel\Nova\Metrics\ValueResult;
+
+class ActiveStudentsInMajor extends Value
+{
+    /**
+     * Calculate the value of the metric.
+     */
+    public function calculate(NovaRequest $request): ValueResult
+    {
+        $count = Major::where('id', $request->resourceId)->get()->first()->members()->active()->count();
```

Comment:
You don't need to fix it but `::find(id)` is an alias of `::where('id', id)->get->first()`. https://laravel.com/docs/5.8/eloquent#retrieving-single-models